nbgrader/tests/apps/test_nbgrader_validate.py

```diff
@@ -85,13 +89,15 @@ def test_validate_unchanged_ignore_checksums(self):
             "validate", "submitted-unchanged.ipynb",
             "--Validator.ignore_checksums=True"
         ], stdout=True)
-        assert output.splitlines()[0] == "VALIDATION FAILED ON 1 CELL(S)! If you submit your assignment as it is, you WILL NOT"
+        assert output.splitlines()[
+                   0] == "VALIDATION FAILED ON 1 CELL(S)! If you submit your assignment as it is, you WILL NOT"
 
     def test_locked_cell_changed(self):
         """Does the validate fail if a locked cell has changed?"""
         self._copy_file(join("files", "submitted-locked-cell-changed.ipynb"), "submitted-locked-cell-changed.ipynb")
         output = run_nbgrader(["validate", "submitted-locked-cell-changed.ipynb"], stdout=True)
-        assert output.splitlines()[0] == "THE CONTENTS OF 2 TEST CELL(S) HAVE CHANGED! This might mean that even though the tests"
+        assert output.splitlines()[
+                   0] == "THE CONTENTS OF 2 TEST CELL(S) HAVE CHANGED! This might mean that even though the tests"
```

Comment:
```suggestion
        assert (
            output.splitlines()[0]
            == "THE CONTENTS OF 2 TEST CELL(S) HAVE CHANGED! This might mean that even though the tests"
        )
```